src/frontend/app/store/actions/create-service-instance.actions.ts

```diff
@@ -0,0 +1,42 @@
+import { Action } from '@ngrx/store';
+
+import { GitAppDetails, SourceType } from '../types/deploy-application.types';
+import { GitBranch, GithubCommit } from '../types/github.types';
+import { PaginatedAction } from '../types/pagination.types';
+import { IRequestAction } from '../types/request.types';
+import { githubBranchesSchemaKey, githubCommitSchemaKey } from '../helpers/entity-factory';
+
+export const SET_SERVICE_PLAN = '[Create SI] Set Plan';
+export const SET_ORG = '[Create SI] Set Org';
+export const SET_SPACE = '[Create SI] Set Space';
+export const SET_CREATE_SERVICE_INSTANCE = '[Create SI] Set All';
+export const SET_APP = '[Create SI] Set App';
+export const SET_SERVICE_INSTANCE_GUID = '[Create SI] Set Service Instance Guid';
+
+export class SetServicePlan implements Action {
+  constructor(public servicePlanGuid: string) { }
+  type = SET_SERVICE_PLAN;
+}
+export class SetOrg implements Action {
```

Comment:
`SetOrg`, `SetSpace` and `SetApp` are all a bit generic too. If they're specifically for service instance state they should be named as such.